sections/featured-product.liquid

```diff
@@ -30,7 +30,7 @@
   <link id="ModelViewerOverride" rel="stylesheet" href="{{ 'component-model-viewer-ui.css' | asset_url }}" media="print" onload="this.media='all'">
 {%- endif -%}
 
-<section class="{% if section.settings.secondary_background %} background-secondary{% endif %} color-{{ section.settings.color_scheme }} gradient">
+<section class="{% if section.settings.secondary_background %}background-secondary{% else %}gradient{% endif %} color-{{ section.settings.color_scheme }}">
```

Comment:
There are still a few things to figure out I think with how `show seconday background` should work with the color scheme chosen 🤔 
[screenshot](https://screenshot.click/22-01-cphvv-ks1yh.png)

Reply:
Alright, stuck the class further down the chain - it was overriding the foreground colour with white which was causing the issue. 🤦

Reply:
I see - okay, I think that's the behaviour we had before because it was giving it a white background when the text was white.  I'll do it that way and we can test to ensure that it's what we're looking for.

For now, we can keep the secondary background because adding a second picker adds complexity.

Reply:
Okay, I think I've got it working the way we expect it to.

1. If secondary background is not selected, it gets a full screen background with the same colour as the internal container
2. If it is selected, the text colour is taken (either the actual text colour for background, the background colour for inverse, or the button outline colour for accent)

Should be working like that now - I also fixed the gradient so that it works in both cases.